mava/configs/system/rec_iql.yaml

```
@@ -0,0 +1,33 @@
+# --- Defaults REC-IDQN ---
+seed: 1
+n_envs: 32 # Number of environments to run in parallel (vmapped).
+
+# --- Agent observations ---
+add_agent_id: True
+
+# --- RL hyperparameters ---
+# step related
+total_timesteps: 20_000_000
```

Also can we add `num_updates` here + the total_timesteps
NIT:
```suggestion
total_timesteps: 2e7
```